components/map/overview.md

```diff
@@ -0,0 +1,243 @@
+---
+title: Overview
+page_title: Map Overview
+description: Discover the Blazor Map and explore the examples.
+slug: components/map/overview
+tags: telerik,blazor,map,overview
+published: True
+position: 0
+---
+
+# Map Overview
+
+The Map displays geospatial information organized in layers. It is supported for both desktop and mobile devices.
```

Comment:
I suppose that it is implied that we support both desktop and mobile devices. Most often, we go the other way around - we add a note if a specific browser, environment, etc. is not supported.

Reply:
Removed that line.